mteb/tasks/BitextMining/multilingual/NusaXBitextMining.py

```diff
@@ -0,0 +1,64 @@
+from __future__ import annotations
+
+from mteb.abstasks.TaskMetadata import TaskMetadata
+
+from ....abstasks import AbsTaskBitextMining, CrosslingualTask
+
+_LANGUAGES = {
+    "eng-ace": ["eng-Latn", "ace-Latn"],
+    "eng-ban": ["eng-Latn", "ban-Latn"],
+    "eng-bbc": ["eng-Latn", "bbc-Latn"],
+    "eng-bjn": ["eng-Latn", "bjn-Latn"],
+    "eng-bug": ["eng-Latn", "bug-Latn"],
+    "eng-ind": ["eng-Latn", "ind-Latn"],
+    "eng-jav": ["eng-Latn", "jav-Latn"],
+    "eng-mad": ["eng-Latn", "mad-Latn"],
+    "eng-min": ["eng-Latn", "min-Latn"],
+    "eng-nij": ["eng-Latn", "nij-Latn"],
+    "eng-sun": ["eng-Latn", "sun-Latn"],
+}
+
+
+class NusaXBitextMining(AbsTaskBitextMining, CrosslingualTask):
+    metadata = TaskMetadata(
+        name="NusaXBitextMining",
+        dataset={
+            "path": "gentaiscool/bitext_nusax",
+            "revision": "fba4f2cfe2592641056f7a274c9aa8453b27a4a8",
+        },
+        description="NusaX is a parallel dataset for machine translation and sentiment analysis on 11 Indonesia languages and English.",
+        reference="https://huggingface.co/datasets/indonlp/NusaX-senti/",
+        type="BitextMining",
+        category="s2s",
+        eval_splits=["train"],
+        eval_langs=_LANGUAGES,
+        main_score="f1",
+        date=("2023-01-01", "2023-03-01"),
```

Comment:
This is intended to be data creation dates (when it was written). Can be an estimate

Reply:
I just updated the date range to `date=("2021-08-01", "2022-07-01")`

Reply:
That was the time when we collected the dataset